### ISSUE TITLE
Unable to bind tf.keras.* classes 

### ISSUE THREAD

**Issue by kuldeep-techchefs** [google/gin-config#19]
config.gin

build_model.optimizer = @tf.keras.optimizers.Adam()
build_model.loss_fn = @tf.keras.losses.sparse_categorical_crossentropy

---------------------------------------------------------------------------------------------------

@gin.configurable
 def build_model(self, optimizer, loss_fn):
        self.model.compile(optimizer=optimizer,
              loss=loss_fn,
              metrics=['accuracy'])

---------------------------------------------------------------------------------------------------

Getting following error

No configurable matching reference '@tf.keras.optimizers.Adam()'.
  In file "../configs/config.gin", line 4
    build_model.optimizer = @tf.keras.optimizers.Adam()

---------------------------------------------------------------------------------------------------
By the way it works fine if I replace @tf.keras.optimizers.Adam() with @tf.train.AdamOptimizer()

**Comment by sguada** [comment]
To make any external Class or function configurable you need to use `gin.external_configurable` as done in [external_configurables](https://github.com/google/gin-config/blob/master/gin/tf/external_configurables.py#L72)

`gin.external_configurable(tf.keras.optimizers.Adam, module='tf.keras.optimizers')`

**Comment by kuldeep-techchefs** [comment]
Thanks @sguada seems working for Optimizer
